SDL/EndcapGeometry.h

```diff
@@ -9,7 +9,11 @@
 #include <vector>
 #include <stdexcept>
 
+#ifdef LST_STANDALONE
 #include "Constants.h"
+#else
+#include <RecoTracker/LSTCore/interface/alpaka/Constants.h>
```

Comment:
```suggestion
#include "RecoTracker/LSTCore/interface/alpaka/Constants.h"
```
will be needed after the files are actually in CMSSW (the rule/style is to use `<>` for system and external includes

Reply:
Oh yeah sorry, I'll fix that